packages/openapi3/src/openapi.ts

```diff
@@ -1269,19 +1261,10 @@ function createOAPIEmitter(
     }
     const contentType = contentTypes[0];
 
-    switch (contentType) {
-      case "text/plain":
-        return schema.type === "string" || schema.type === "number";
-      case "application/octet-stream":
-        return (
-          isRawBinarySchema(schema) ||
-          (schema.type === "array" && !!schema.items && isRawBinarySchema(schema.items as any))
-        );
-      case "application/json":
-        return schema.type === "object";
-    }
-
-    return false;
+    return contentType === "text/plain";
```

Comment:
I don't think this change is correct, the openapi3 spec does define those default here https://spec.openapis.org/oas/v3.0.3.html#special-considerations-for-multipart-content

Reply:
Thanks everyone for helping me get to the bottom of this! I believe the code is doing the right thing at it is. I wasn't aware of all the conditional defaults here.